core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -128,44 +215,16 @@ class BpmBillingProjectLifecycle(
           )
         case _ => ()
       }
-      _ <- billingRepository.getLandingZoneId(projectName).map {
+      _ <- billingRepository.getLandingZoneId(projectName).flatMap {
         case Some(landingZoneId) =>
-          val landingZoneResponse = blocking {
-            // Note that this actually just starts a landing zone deletion job (and thus returns quickly).
-            // We are not attempting to ensure that the landing zone deletion completes successfully.
-            workspaceManagerDAO.deleteLandingZone(UUID.fromString(landingZoneId), ctx)
-          }
-          Option(landingZoneResponse.getErrorReport) match {
```

Comment:
The refactoring introduced a change in behavior here-- we now don't throw a `BillingProjectDeletionException` if we get back an error report from the landing zone deletion, and instead just log it and proceed with deleting the billing profile.